Tools/CISupport/ews-build/steps.py

```diff
@@ -543,9 +546,11 @@ def start(self):
         d.addCallback(lambda res: shell.ShellCommand.start(self))
 
     def hideStepIf(self, results, step):
-        return results == SUCCESS and self.getProperty('sensitive', False)
+        return not self.doStepIf(None) or (results == SUCCESS and self.getProperty('sensitive', False))
 
     def getResultSummary(self):
+        if self.results == SKIPPED:
+            return {'step': 'No patch to apply'}
```

Comment:
Can reword something like: 'Skipping applying patch' or "Skipping applying patch since patch_id isn't provided" or "Skipping applying patch in PR workflow".

'No patch to apply' feels like indicating an error in which patch was expected but not found.